operations/mimir-mixin-compiled/alerts.yaml

```diff
@@ -68,7 +68,7 @@ groups:
       message: |
         There are {{ $value }} queued up queries in {{ $labels.cluster }}/{{ $labels.namespace }} {{ $labels.job }}.
     expr: |
-      sum by (cluster, namespace, job) (cortex_query_frontend_queue_length) > 1
+      sum by (cluster, namespace, job) (min_over_time(cortex_query_frontend_queue_length[1m])) > 1
```

Comment:
In the issue I was proposing to change this to `> 0`.

Reply:
Oh sure, fixed in https://github.com/grafana/mimir/pull/2630/commits/3a9685b6fda55c00a74cbb6803f773b3cbc99ffd